.github/workflows/publish_to_maven.yml

```
@@ -0,0 +1,55 @@
+name: Publish to Maven
+on: [pull_request, workflow_dispatch]
```

```suggestion
on: [workflow_dispatch]
```
Don't want to release on every pull request

---

@sarahkoop I had to add `pull_request` because of a bug in github actions. If i only add `workflow_dispatch` it wont appear, unless that file is already on `main` branch. I was going to delete them once this PR is approved

---

Maybe I'm confused - it looks like the release github workflow is executing successfully, what is preventing these artifacts from being published automatically currently?

---

its like this: github actions has a bug. If your `.yml` is not on your default branch (in our case `main` branch) and your workflow ONLY has `workflow_dispatch`, you wont be able to see it in the github UI. So, in order for me to test it, i had to add `pull_request`, and only then the workflow appeared in the github UI. Before merging it to main, i'm going to delete `pull_request` tag, but until i get all the approvals, i need to leave it there so that i can test the workflow. Does that make sense?